### ISSUE TITLE
Head tracking issues in teleportation-related levels (Unreal 4.12.2)

### ISSUE THREAD

**Issue by leestar** [mitchemmc/VRContentExamples#1]
I'm noticing weird head tracking issues in the teleportation-related levels, when running on Unreal 4.12.2. There seems to be a slow juddering effect, as if the camera isn't properly interpreting HMD movement. I've only noticed in the simple teleportation level and the advanced teleportation level.

To reproduce: open the simple teleportation level, and add a simple cube mesh to the scene. Give it a texture like grass so the effect is more noticeable. Then run a VR preview, walk up to the mesh, and move your head towards and away from it, you should immediately see what I mean.

**Comment by DanielHofmeier** [comment]
I can confirm this issue. The problem appears also on 4.12.3

**Comment by mitchemmc** [comment]
I believe this would be an issue with the now proper implementation of head position late update in 4.12 and not hitting 90fps, I just pushed a new update to master that fixes perf issues so give that a try and let me know how it goes. Also the advanced teleportation level is old and has now been replaced with the trace interaction level

Edit: just for clarity I didn't see any tracking issues in my testing but I relatively immune to seeing them

**Comment by leestar** [comment]
Quick response! Unfortunately though I just tested and am still noticing it, in 4.12.2

It's a little hard to see if your mesh doesn't have a patterned material - try something with lots of detail like  the grass from the starter content. Maybe "juddering" is the wrong word - what it seems to be is some kind of "swaying" effect where the mesh moves left and right as you move your head towards and away.

Btw if it helps - I tried replacing the SimpleTelePawn with the WorldInteractionPawn from the VR_World_Interaction level, and the swaying effect went away, so perhaps it is due to some component or Blueprint logic in SimpleTelePawn. I can also confirm that this problem doesn't arise in 4.11.
